app/hotels/src/search/__tests__/SearchQueryHelpers.test.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+import { getSearchQueryParams, hasCoordinates } from '../SearchQueryHelpers';
+import { sanitizeDate } from '../../GraphQLSanitizers';
+
+const defaultSearch = {
+  checkin: new Date(1),
+  checkout: new Date(2),
+  roomsConfiguration: { adultsCount: 1, children: [] },
+};
+
+const defaultCoordinates = {
+  latitude: 3,
+  longitude: 4,
+};
+
+describe('SearchQueryHelpers', () => {
+  describe('hasCoordinates', () => {
+    it('returns true when coordinates are passed', () => {
+      expect(
+        hasCoordinates({
+          latitude: 34.3,
+          longitude: 12.12312,
+        }),
+      ).toBe(true);
+    });
+
+    it('returns false if latitude is not defined', () => {
+      expect(
+        // $FlowExpectedError: Expect error since it breaks with flow type
```

Comment:
Of course it breaks with flow type - this is why you need to use the `// $FlowExpectedError` comment. It's usually better to explain why. Especially why you decided not to fix it here or why you are OK with this hack.

Look at this from my point of view - I don't know what is the reason for this. So I have to remove the comment, run Flow and only after that I'll realize it's because `hasCoordinates` accepts object with all fields and you are intentionally not passing them (you can improve the readability with `latitute: undefined` to make it really obvious).

Just a tip - think about it... :)